massa-execution-exports/Cargo.toml

```diff
@@ -18,7 +18,9 @@ massa_storage = { path = "../massa-storage" }
 massa_final_state = { path = "../massa-final-state" }
 massa_ledger_exports = { path = "../massa-ledger-exports", optional = true }
 parking_lot = { version = "0.12", features = ["deadlock_detection"], optional = true }
+massa-sc-runtime = { git = "https://github.com/massalabs/massa-sc-runtime", branch = "add_gas_calibration_results" }
 
 # for more information on what are the following features used for, see the cargo.toml at workspace level
 [features]
+gas_calibration = ["massa_ledger_exports/testing", "parking_lot"]
```

Comment:
Is there any point of restricting parking_lot for some features as it is widely used in our code?

Reply:
It shouldn't be used in exports in a normal case as exports crate only expose data structures so I limit parking_lot only for this feature